### PR TITLE
Added apache integration

### PR DESCRIPTION
**Pull request by 72nomada**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-threatintel/issues/47|

Apache HTTP integration.

### CODE REVIEW

File: src/engine/ruleset/integrations/apache-http/changelog.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+items:
+  - version: 1.0.0-dev
+    short: Created integration for apache http server
+    pr: "[#PR0000](#)"
```
Comment (JcabreraC) [inline src/engine/ruleset/integrations/apache-http/changelog.yml:4]: Add the PR.

File: src/engine/ruleset/integrations/apache-http/decoders/apache-access.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    - "2.4.16"
+  author:
+    name: Wazuh Inc.
+    email: info@wazuh.com
+    date: 2022-11-15
+  references:
+    - https://httpd.apache.org/docs/2.4/logs.html
+
+check:
+  - event.original: +regex_match/\S+\s-\s\S+\s\[
```
Comment (JcabreraC) [inline src/engine/ruleset/integrations/apache-http/decoders/apache-access.yml:22]: Leave a comment with the `check` that would be used with the configuration labels. This way, if the integration model we want is applied, the change can be automatic.

```
#TODO: Once the events arrive tagged, uncomment these lines below and remove the above regex
  # - event.module: apache-http
  # - event.dataset: apache-access
  ```

File: src/engine/ruleset/integrations/apache-http/decoders/apache-error.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  author:
+    name: Wazuh Inc.
+    email: info@wazuh.com
+    date: 2022-11-15
+  references:
+    - https://httpd.apache.org/docs/2.4/logs.html
+    - https://httpd.apache.org/docs/2.4/custom-error.html
+
+check:
+  - event.original: +starts_with/[
```
Comment (JcabreraC) [inline src/engine/ruleset/integrations/apache-http/decoders/apache-error.yml:23]: Leave a comment with the `check` that would be used with the configuration labels. This way, if the integration model we want is applied, the change can be automatic.

```
#TODO: Once the events arrive tagged, uncomment these lines below and remove the above `event.original`
  # - event.module: apache-http
  # - event.dataset: apache-error
  ```

**Review by JcabreraC — CHANGES_REQUESTED** [review]

File: src/engine/ruleset/integrations/apache-http/changelog.yml (hunk 1 above)
Comment (72nomada) [reply src/engine/ruleset/integrations/apache-http/changelog.yml]: [#16910](https://github.com/wazuh/wazuh/pull/16910)

both changelog and readme updated

File: src/engine/ruleset/integrations/apache-http/decoders/apache-access.yml (hunk 2 above)
Comment (72nomada) [reply src/engine/ruleset/integrations/apache-http/decoders/apache-access.yml:22]: done

File: src/engine/ruleset/integrations/apache-http/decoders/apache-error.yml (hunk 3 above)
Comment (72nomada) [reply src/engine/ruleset/integrations/apache-http/decoders/apache-error.yml:23]: done

**Review by JcabreraC — APPROVED** [review]
LGTM !

**Review by JavierBejMen — CHANGES_REQUESTED** [review]
schemas/fields.json should not be modified

**Review by JavierBejMen — APPROVED** [review]
LGTM

**Pull request merged**